pytorch_lightning/plugins/training_type/ipu.py

```diff
@@ -27,6 +29,7 @@
 from pytorch_lightning.utilities import _IPU_AVAILABLE, _POPTORCH_AVAILABLE
 from pytorch_lightning.utilities.apply_func import apply_to_collection
 from pytorch_lightning.utilities.cloud_io import get_filesystem
+from pytorch_lightning.utilities.data import _get_dataloader_init_kwargs
```

Comment:
```suggestion
from pytorch_lightning.utilities.data_loading import _get_dataloader_init_kwargs
```

Reply:
Just a suggestion, I am open to both :)

Reply:
data_loading does not exist in the utilities folder and data.py was already there. I wish not to do file renamings in this PR since this will require adding deprecation boilerplate